gridspeccer/cli.py

```diff
@@ -47,6 +47,13 @@ def plot():
         type=str,
         default="../fig",
     )
+    parser.add_argument(
+        "--independent-plots-filetype",
+        help="filetype of independent plots. those are queried from additional"
+        "gridspecs that are then plotted independently",
+        type=str,
+        default="None",
```

Comment:
Why use the case-sensitive string `"None"` here? The argument could simply be made optional so that you can just check for `None`.